NetCord/User.cs

```diff
@@ -2,43 +2,230 @@
+/// </remarks>
 public partial class User(JsonModels.JsonUser jsonModel, RestClient client) : ClientEntity(client), IJsonModel<JsonModels.JsonUser>
 {
     JsonModels.JsonUser IJsonModel<JsonModels.JsonUser>.JsonModel => _jsonModel;
     private protected readonly JsonModels.JsonUser _jsonModel = jsonModel;
 
+    /// <summary>
+    /// The user's ID.
+    /// </summary>
+    /// <remarks>
+    /// Requires the <c>identify</c> OAuth2 scope.
+    /// </remarks>
     public override ulong Id => _jsonModel.Id;
+
+    /// <summary>
+    /// The user's username, not unique across the platform. Restrictions:
+    /// </summary>
+    /// <remarks>
+    /// <list type="bullet">
+    ///     <item>
+    ///         <description>
+    ///         Must be between 2 and 32 characters long.
+    ///         </description>
+    ///     </item>
+    ///     <item>
+    ///         <description>
+    ///         Usernames cannot contain the following substrings: <c>@</c>, <c>#</c>, <c>:</c>, <c>```</c>, <c>discord</c>.
+    ///         </description>
+    ///     </item>
+    ///     <item>
+    ///         <description>
+    ///         Usernames cannot be: <c>everyone</c> or <c>here</c>.
+    ///         </description>
+    ///     </item>
+    /// </list>
+    /// Requires the <c>identify</c> OAuth2 scope.
+    /// </remarks>
     public string Username => _jsonModel.Username;
+
+    /// <summary>
+    /// The user's Discord-tag.
+    /// </summary>
+    /// <remarks>
+    /// Requires the <c>identify</c> OAuth2 scope.
+    /// </remarks>
     public ushort Discriminator => _jsonModel.Discriminator;
+
+    /// <summary>
+    /// The user's display name, if it is set. For bots, this is the application name. Restrictions:
+    /// </summary>
+    /// <remarks>
+    /// <list type="bullet">
+    ///     <item>
+    ///         <description>
+    ///         Must be between 1 and 32 characters long.
+    ///         </description>
+    ///     </item>
+    ///     <item>
+    ///         <description>
+    ///         Some zero-width and non-rendering characters are limited.
+    ///         </description>
+    ///     </item>
+    ///     <item>
+    ///         <description>
+    ///         Names are sanitized and trimmed of leading, trailing, and excessive internal whitespace.
+    ///         </description>
+    ///     </item>
+    /// </list>
+    /// Requires the <c>identify</c> OAuth2 scope.
+    /// </remarks>
     public string? GlobalName => _jsonModel.GlobalName;
+
+    /// <summary>
+    /// The user's avatar hash.
+    /// </summary>
+    /// <remarks>
+    /// Requires the <c>identify</c> OAuth2 scope.
+    /// </remarks>
     public string? AvatarHash => _jsonModel.AvatarHash;
+
+    /// <summary>
+    /// Whether the user belongs to an OAuth2 application.
```

Comment:
I feel like that this doc could be improved. I don't think that it should reference OAuth2. Maybe `just belongs too an application`

Reply:
Simplified it.